### PR TITLE
chore: unify blueprint workflows

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

File: .github/workflows/blueprint.yml [hunk 1]
```diff
@@ -62,14 +86,26 @@ jobs:
...
 
-      - name: Build blueprint
+      - name: Check for `docs` folder # this is meant to detect a Jekyll-based website
+        id: check_docs
+        run: |
+          if [ -d docs ]; then
+            echo "The 'docs' folder exists in the repository."
+            echo "DOCS_EXISTS=true" >> $GITHUB_ENV
+          else
+            echo "The 'DOCS' folder does not exist in the repository."
```
Comment (Ruben-VandeVelde) [inline .github/workflows/blueprint.yml:99]:
```suggestion
            echo "The 'docs' folder does not exist in the repository."
```

File: .github/workflows/blueprint.yml [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+    branches:
+      - main
+    paths:
+      - '**/*.lean'
+      - '**/blueprint.yml'
+      - 'blueprint/**'
+      - 'docs/**'
+      - 'lean-toolchain'
+      - 'lakefile.toml'
+      - 'lake-manifest.json'
```
Comment (Ruben-VandeVelde) [inline .github/workflows/blueprint.yml:25]: Does this exclude anything?

**Review by Ruben-VandeVelde — APPROVED** [review]
Seems okay, I think

File: .github/workflows/blueprint.yml (hunk 2 above)
Comment (pitmonticone) [reply .github/workflows/blueprint.yml:25]: Yes, everything not lean or blueprint related.

**Pull request merged**